htdocs/user/class/user.class.php

```diff
@@ -328,8 +328,7 @@ function fetch($id='', $login='', $sid='', $loadpersonalconf=0, $entity=-1)
 				// fetch optionals attributes and labels
 				require_once(DOL_DOCUMENT_ROOT.'/core/class/extrafields.class.php');
 				$extrafields=new ExtraFields($this->db);
-				$extralabels=$extrafields->fetch_name_optionals_label($this->table_element,true);
-				$this->fetch_optionals($this->id,$extralabels);
+				$this->fetch_optionals($this->id);
```

Comment:
Usage without $extralabels parameter needs to have $extrafields->attributes loaded.
But if fetch_name_optionals_label is not called, array is not loaded so we can't call without a parameter